website/pages/submit-project.html

```
@@ -1,165 +1,286 @@
-                        <textarea id="challenges" name="challenges"
-                                  placeholder="Describe any challenges you encountered and how you solved them" rows="3"></textarea>
-                    </div>
-
-                    <div class="form-group">
-                        <label for="futureImprovements">Future Improvements</label>
-                        <textarea id="futureImprovements" name="futureImprovements"
-                                  placeholder="What features or improvements do you plan to add?" rows="3"></textarea>
-                    </div>
+            <div class="form-group">
+                <label>Project Title *</label>
+                <input type="text" placeholder="Enter your project title">
```

HTMLHint (1.9.0)

[warning] The empty tag : [ input ] must be self closed.

(tag-self-close)